### DISCUSSION TITLE
There appears to be an inconsistency with how commit timestamps are displayed for February 29, suggesting a potential leap year handling bug.

### DISCUSSION THREAD

**Discussion by qiaoqiaoLF** [community/community#115491] [Repositories]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/57006825/ed1ff627-4834-4348-8b37-25fae4a0fa0d)
I pushed a commit to my private repo BiT on February 29, but when I checked it today on March 31, the commit date seemed incorrect (telling me it was made yesterday)

**Comment by elbekgiyozov** [comment]
(please mark as answer 🙏🙏🙏)
